scripts/pylint_extensions_test.py

```diff
@@ -121,36 +147,319 @@ def test_finds_hanging_indent(self):
         ):
             temp_file.close()
 
-        node_no_err_message = astroid.scoped_nodes.Module(
+        node_with_no_error_message = astroid.scoped_nodes.Module(
             name='test',
             doc='Custom test')
 
         temp_file = tempfile.NamedTemporaryFile()
         filename = temp_file.name
         with open(filename, 'w') as tmp:
             tmp.write(
-                """master_translation_dict = json.loads(
+                """\"""Some multiline
```

Comment:
Why this addition? What does it add to the existing test? This sample docstring is not associated with any function and has no args or returns.

Reply:
This adds a multi-line docstring and a comment to the existing file. This is required to test [this](https://github.com/oppia/oppia/blob/055481d4efd42f5c1771b25ce32e65e9a5cab77a/scripts/pylint_extensions.py#L170) and the following lines.

> This sample docstring is not associated with any function and has no args or returns.

Is it necessary for a docstring to be associated with only a function? I mean, this docstring denotes the file overview (which is present in all files in our code-base)